xsd/netex-nl-basic.xsd

```diff
@@ -208,7 +205,7 @@ Bij een object in een assignment is attribuut 'order' verplicht maar niet (altij
 		</xsd:annotation>
 		<xsd:attribute name="id" type="ObjectIdType" use="required" form="unqualified"/>
 		<xsd:attribute name="modification" type="ModificationEnumeration" use="optional" default="new" form="unqualified"/>
-		<xsd:attribute name="version" type="VersionIdType" use="optional" default="any" form="unqualified"/>
+		<xsd:attribute name="version" type="VersionIdType" use="required" form="unqualified"/>
```

Comment:
Bewuste keuze default hier weg te halen?

Reply:
ja: als het attribuut required is, overschrijf je altijd de default waarde, dus is die niet nodig